### DISCUSSION TITLE
ASA passive mod support

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#718] [Suggestions]
It would be greate to add an option to set a mod to passive. What this needs to do is 2 things.

1. Set the mod(s) to first in the load order.
2. Add the line -passivemods=ModID in the command line section. (Think its the end section on GSA)

Christmas example.
Example: -mods=927090,ModID,ModID -passivemods=927090

https://discord.com/channels/315049169071112193/1166795937453199410/1194754539878883388

![image](https://github.com/gameserverapp/Platform/assets/749937/d22e0abf-0026-4661-891f-40dbe138b284)

**Comment by mvaessen** [comment]
this can be managed through the config template `end` section. You probably already knew that, but I figure i mention it for other folks.

As for the passive mods: since this is very specific to ark, and isnt used a whole lot, I don't think it's worth the development time.

**Comment by birdman3131** [comment]
I was under the impression that it could not be used with the End section if you wanted GSA to be able to manage mods. If this is not the case then it is probably fine as is. (Assuming you can reorder mods in GSA for ASA. I recall that being added but my memory if fuzzy on it.)

Edit: As far as I know its been done on all the official event mods sometimes taking months for it to come to the base game.

**Reply by mvaessen** [reply]
The "passivemods" shouldnt cause any issues when used in the end section. Its only the -mods= that causes issues with gsa mod management afaik

**Reply by birdman3131** [reply]
I bet I know why I am not recalling it correctly. There was a point when ASA came out that you could not reorder mods on GSA for ASA. I'll bet passive mods came out before you added that ability and I am just remembering from back then. I do recall you adding it fairly soon after launch but memory is fuzzy as to which was first in the timeline.
